### PR TITLE
Improved prompt and added voice feedback.

### PR DESCRIPTION
**Pull request by davidzhao**

### CODE REVIEW

**Review by keepingitneil — APPROVED** [review]

File: examples/detection/detection.py [hunk 1]
```diff
@@ -142,11 +177,21 @@ async def detect(self, frame: rtc.VideoFrame):
...
-    async def publish_video(self):
-        track = rtc.LocalVideoTrack.create_video_track("agent-video", self.video_out)
-        options = rtc.TrackPublishOptions()
-        options.source = rtc.TrackSource.SOURCE_CAMERA
-        await self.ctx.room.local_participant.publish_track(track, options)
+    async def send_audio_stream(
+        self, tts_stream: AsyncIterable[agents.tts.SynthesisEvent]
+    ):
+        async for e in tts_stream:
+            if e.type == agents.tts.SynthesisEventType.STARTED:
```
Comment (theomonnom) [inline examples/detection/detection.py:184]: Conditions aren't necessary here, but the streams of TTS still need discussion anyway

File: examples/detection/detection.py [hunk 2]
```diff
@@ -71,18 +99,25 @@ def on_chat_received(self, message: rtc.ChatMessage):
...
-        track: rtc.Track,
-        publication: rtc.TrackPublication,
-        participant: rtc.RemoteParticipant,
-    ):
-        self.ctx.create_task(self.process_track(track))
+    async def end_session_after(self, duration: int):
+        await asyncio.sleep(duration)
+        await self.send_chat_and_voice(BYE_MESSAGE)
+        await asyncio.sleep(3)
+        self.ctx.room.disconnect()
```
Comment (theomonnom) [inline examples/detection/detection.py:111]:
```suggestion
        self.ctx.disconnect()
```

**Review by theomonnom — APPROVED** [review]
lg

File: examples/detection/detection.py [hunk 3]
```diff
@@ -162,7 +207,5 @@ async def job_request_cb(job_request: agents.JobRequest):
             auto_disconnect=agents.AutoDisconnect.DEFAULT,
         )
 
-    worker = agents.Worker(
-        request_handler=job_request_cb, worker_type=agents.JobType.JT_ROOM
-    )
+    worker = agents.Worker(request_handler=job_request_cb)
```
Comment (theomonnom) [inline examples/detection/detection.py:210]: nit: you can directly do that
```suggestion
    worker = agents.Worker(job_request_cb)
```

File: examples/detection/detection.py [hunk 4]
```diff
@@ -36,26 +47,43 @@ async def create(cls, ctx: agents.JobContext):
...
-                    examples_to_include=["cell phone"],
-                    examples_to_exclude=[],
-                    detection_threshold=0.2,
-                )
-            ]
-        )
-        self.video_out = rtc.VideoSource(width=640, height=480)
+        self.detector: Optional[Detector] = None
+        self.tts_plugin = TTS(model_id="eleven_turbo_v2")
+        self.video_out = rtc.VideoSource(_OUTPUT_WIDTH, _OUTPUT_HEIGHT)
```
Comment (theomonnom) [inline examples/detection/detection.py:52]: I like this naming

**Pull request merged**